assets/js/trackhar.tsx

```diff
@@ -0,0 +1,264 @@
+                                                                ) : r.reasoning.endsWith('.md') ? (
+                                                                    <a
+                                                                        href={absUrl(
+                                                                            'research/' +
+                                                                                r.reasoning
+                                                                                    .replace(/\/([^/]+)$/, '#$1')
+                                                                                    .replace(/.md$/, '')
+                                                                        )}
+                                                                        target="_blank">
+                                                                        <img
+                                                                            src="/svg/information.svg"
+                                                                            class="inline-icon icon-information"
+                                                                            title={_t(
+                                                                                'tracker-single-transmitted-data-table-reasoning-link-desc',
+                                                                                'hugo'
+                                                                            )}
+                                                                            alt={_t(
+                                                                                'tracker-single-transmitted-data-table-reasoning-link-desc',
+                                                                                'hugo'
+                                                                            )}
+                                                                        />
+                                                                    </a>
+                                                                ) : (
+                                                                    <></>
+                                                                )}
+                                                            </code>
+                                                        </td>
+                                                        <td>
+                                                            <code>{r.value}</code>
+                                                        </td>
+                                                    </tr>
+                                                ))}
+                                            </tbody>
+                                        </table>
+                                    </>
+                                ) : (
+                                    <em>{t('unsupported-endpoint')}</em>
```

Comment:
I don't understand your question. This text is shown for requests that are not matched by any adapter if you disable the "Hide unmatched requests" checkbox. Those endpoints _are_ unsupported.

Reply:
Yes, but if an endpoint is contacted, but no data (other than IP) is transmitted, do we consider it supported?

Reply:
TrackHAR will return `[]` (truthy in JS) in that case, or `undefined` if no adapter or indicators matched.

Reply:
But is this the behaviors we want? Right now, it looks like this if the adapter didn’t match any transmitted data:

![image](https://github.com/tweaselORG/tracker-wiki/assets/4049135/ba82cf3a-63df-4a2b-b3fe-d3ba3538ba5b)

I believe we should just hide these requests, since we can not claim any transmissions there.

Reply:
Oh, I see. Um, but "wasn't matched by any adapter" isn't true, either. I guess you'd have to implement a third case, "matched but no transmissions detected".

Reply:
Will do.